src/main/kotlin/racingcar/view/ResultView.kt

```diff
@@ -0,0 +1,21 @@
+package racingcar.view
+
+import racingcar.domain.Car
+
+object ResultView {
+
+    private const val LOOP_START_NUMBER = 0
+
+    fun carPositionPrints(car: List<Car>) {
+        car.forEach {
+            carPositionPrint(it)
+        }
+    }
+
+    private fun carPositionPrint(car: Car) {
+        (LOOP_START_NUMBER..car.carPosition()).forEach { _ ->
```

Comment:
[CharSequence.repeat](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/repeat.html)를 활용하면 다음과 같이도 작성이 가능해요! 

```suggestion
        println("-".repeat(car.carPosition()))
```

Reply:
코틀린 좋은 API들 정말 많이 지원하네요!
좋은 피드백 항상 감사드려요! 🙇🏻‍♂️